### PR TITLE
Add BlipIcon Id's (Intellisense)

### PR DESCRIPTION
**Pull request by LukeSeers**

### CODE REVIEW

File: SlipeServer.Server/Elements/Enums/BlipType.cs [hunk 1]
```diff
@@ -6,69 +6,69 @@ namespace SlipeServer.Server.Elements
...
+        Barbers = 7,
+        BigSmoke = 8,
+        Boatyard = 9,
+        BurgerShot = 10,
+        Bulldozer = 11,
+        CatPink = 12,
+        Cesar = 13,
+        Chicken = 14,
+        Cj = 15,
+        Crash1 = 16,
```
Comment (LukeSeers) [inline SlipeServer.Server/Elements/Enums/BlipType.cs:25]: I looked into the MS guidelines regrading having a number in the Enum like: **Crash1**, I could be blind like a bat, but I couldn't find anything mentioning about it. I did notice in your latest branch **(vehicle-sync)** you (@NanoBob) used the Enum like: **CrashOne**


So, what would you like me to use in the future?
1. **Crash1**
2. **CrashOne**

Comment (LukeSeers) [reply SlipeServer.Server/Elements/Enums/BlipType.cs:25]: I should also mention atm I'm just following the syntax of the MTA wiki: https://wiki.multitheftauto.com/wiki/Radar_Blips

**Comment by NanoBob** [comment]
What exactly is the point of adding the numeric values to the enum? The default enum numbering already starts at 0 and increments by one, resulting in the same set of numerical values.

**Comment by LukeSeers** [comment]
Personally I find having the values with the Enums help to keep everything readable. I'm more then happy to close this pull request if you disagree.

**Comment by NanoBob** [comment]
The values of the enum aren't required to understand what the enum does, and since the language already uses the proper values by default I would just omit them here.

**Pull request closed without merging**